topi/include/topi/elemwise.h

```diff
@@ -71,6 +71,29 @@ inline Tensor negative(const Tensor& x,
   }, name, tag);
 }
 
+/*!
+* \brief Returns the sign of the tensor
+*
+* \param x The input tensor
+* \param name The name of the operation
+* \param tag The tag to mark the operation
+*
+* \return A Tensor whose op member is the sign
+*/
+inline Tensor sign(const Tensor& x,
+                   std::string name = "tensor",
+                   std::string tag = kElementWise) {
+  return compute(x->shape, [&](const Array<Var>& i) {
+    Expr zero = make_zero(x->dtype);
+    Expr one = make_const(x->dtype, 1);
+    Expr minus_one = make_const(x->dtype, -1);
+    auto s1 = tvm::ir::Select::make((x(i) < zero), minus_one, x(i));
+    auto s2 = tvm::ir::Select::make((s1 > zero), one, s1);
+    auto s3 = tvm::ir::Select::make((s2 == zero), zero, s2);
```

Comment:
Why make 3 selects instead of

    auto s1 = tvm::ir::Select::make((x(i) < zero), minus_one, zero);
    auto s2 = tvm::ir::Select::make((x(i) > zero), one, s1);
    return s2;

?

Reply:
Yeah that sounds better. I have made these changes.